frontend/src/components/commons/TabNavigation.jsx

```diff
@@ -48,14 +49,16 @@ class TabNavigation extends Component {
   render() {
     return (
       <div>
-        <ul role="tablist" className="nav nav-tabs" style={styles.bootstrapNav}>
-          {this.props.tabSpecs.map((tab, key) => (
-            <span key={tab.id} onClick={() => this.selectTab(tab.id)}>
-              <Tab tabName={tab.tabName} selected={tab.id === this.state.currentTab} />
-              &nbsp;&nbsp;&nbsp;&nbsp;
-            </span>
-          ))}
-        </ul>
+        <nav aria-label={this.props.menuName} role="dialog">
```

Comment:
I'm surprised this would be role dialog? Wouldn't it just be another role nav?

Reply:
role nav or navigation are not detected the narrator at all. Instead it just reads the buttons/tab/etc like a bulleted list. Only dialog seems to work in any browser for some reason.